src/unread/UnreadCards.js

```diff
@@ -51,6 +52,7 @@ export default function UnreadCards(props: Props): Node {
   type Card =
     | UnreadStreamItem
     | {| key: 'private', data: $ReadOnlyArray<React$ElementConfig<typeof PmConversationList>> |};
+
```

Comment:
nit: this change isn't related to what the commit is about

Reply:
What would be the best thing to do about this, should I remove this from the commit using a `git rebase -i` ?

Reply:
Yeah, just remove the change from the commit.